### PR TITLE
#158518449 update passenger profile  

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?

Adds passengers' profile update form for user to change;
 
* Password 
* Email address
* Username

#### Description of Task to be completed?

* Building (without functionality), the UI elements with HTML&CSS. 

#### How the app should be tested
On a webbrowser:

* [Download](https://github.com/artorious/Ride-My-Way/archive/ft-passenger-profile-update-158518449.zip) the repo
* Extract and navigate into the `Ride-My-Way` folder
* in the `UI` folder, open `update_passenger_profile.html` on a browser

On a  terminal (Linux):

* Install and set up git `sudo apt install git` 

* Clone the repository `git clone https://github.com/artorious/Ride-My-Way.git`

* Run `git checkout ft-passenger-profile-update-158518449`

* Run `cd Ride-My-Way/UI/`

* Open `update_passenger_profile.html` on a browser

#### Any background context you want to provide?
The UI has no functionality. 
Developed using HTML and CSS

#### What are the relevant pivotal tracker stories?
[#158518449](https://www.pivotaltracker.com/story/show/158518449)

#### Screenshots 
passupdate

### CODE REVIEW

**Comment by JoshuaOndieki** [comment]
Consider a margin between update and discard.
What if you had the update button to stand out in order to avoid accidental discard activities by the users.

**Comment by Ogollah** [comment]
It looks nice just check on the comments by @JoshuaOndieki

**Comment by LWanjiru** [comment]
Did you make changes as per the recommendations from your peers? @artorious

File: UI/update_passenger_profile.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          <input type="submit" value="Update">       
+        </div>
+      </fieldset>
+    </form>
+  </div>
+  <hr>
+  <footer>&copy;RideMyWay 2018.<br><a href="#">Contact Us.</a></footer>
+</div>
+</body>
+</html>
```
Comment (LWanjiru) [inline UI/update_passenger_profile.html]: add new line

**Review by LWanjiru — CHANGES_REQUESTED** [review]

**Comment by artorious** [comment]
@Ogollah @JoshuaOndieki @LWanjiru I appreciate the input and direction. 
passupdate

**Pull request merged**